docs/about/team/the-software-development-team.md

```diff
@@ -59,3 +59,9 @@ I hope we can get along, and build the technological world of tomorrow !
 ### [Cardin, Thomas](https://www.linkedin.com/in/thomas-cardin/)
 
 Hello, I'm Thomas Cardin, a DevOps Specialist at CFIA. My job is to streamline software development and deployment to improve system reliability and efficiency. I'll enforce security to protect our operations and align with SRE practices to ensure our infrastructure's scalability and continuous service delivery. 
+
+### [Werbrouck, Francois] (https://www.linkedin.com/in/francois-werbrouck-b17296202)
```

Comment:
Same here for the link. It has to be together [text](link)

Reply:
corrected as stated in the other comment about the link